lib/krane/global_deploy_task.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/deploy_task'
+
+module Krane
+  class GlobalDeployTask < KubernetesDeploy::DeployTask
+    def initialize(**args)
+      super(args.merge(allow_globals: true, namespace: 'default', protected_namespaces: ''))
```

Comment:
Hm, it's pretty weird that we have to set a namespace name, especially a real one, on a non-namespaced command. Come to think of it, that may also be an issue with reusing the Resource superclass (though `TaskConfig` may already be the right tool for solving it in that case).

Reply:
Yeah, once https://github.com/Shopify/kubernetes-deploy/pull/569 merges I can take a second look at removing the requirement.